services/apps/alcs/src/alcs/application-decision/application-decision-condition/application-decision-condition.service.ts

```diff
@@ -152,7 +152,7 @@ export class ApplicationDecisionConditionService {
 
     await this.repository.manager.transaction(async (transactionalEntityManager) => {
       await transactionalEntityManager.remove(lots);
-      await transactionalEntityManager.remove(conditions);
```

Comment:
Same here, I'm not sure we should be changing stuff that is already working this way. Soft delete can have some annoying issues.